WSLGd/main.cpp

```diff
@@ -322,22 +322,27 @@ try {
         LOG_ERROR("shared memory ob directory path is not set.");
     }
 
-    // Create a listening vsock to be used for the RDP connection.
-    //
-    // N.B. getsockname is used to get the port assigned by the kernel.
+    // Create a listening vsock in the reserved port range to be used for the RDP connection.
     sockaddr_vm address{};
     address.svm_family = AF_VSOCK;
     address.svm_cid = VMADDR_CID_ANY;
-    address.svm_port = VMADDR_PORT_ANY;
     socklen_t addressSize = sizeof(address);
     wil::unique_fd socketFd{socket(AF_VSOCK, SOCK_STREAM, 0)};
     THROW_LAST_ERROR_IF(!socketFd);
-    auto socketFdString = std::to_string(socketFd.get());
-    THROW_LAST_ERROR_IF(bind(socketFd.get(), reinterpret_cast<const sockaddr*>(&address), addressSize) < 0);
+    unsigned int maxPort = 1024;
```

Comment:
I think it would be nice to have some description here why 1024? or have a define with reasonable friendly name?